assets/js/admin/admin-scripts.js

```diff
@@ -2190,3 +2196,72 @@ function get_url_params() {
 	}
 	return vars;
 }
+
+/**
+ * Run when user click on upload CSV.
+ *
+ * @since 1.8.13
+ */
+function give_on_donation_import_start() {
+    give_on_donation_import_ajax();
+}
+
+/**
+ * Upload CSV ajax
+ *
+ * @since 1.8.13
+ */
+function give_on_donation_import_ajax() {
+    var $form = jQuery('form.tools-setting-page-import');
+
+    var progress = jQuery( $form ).find( '.give-progress' );
```

Comment:
@raftaar1191 No need to wrapper `$form` again with`jQuery`. It is already a `jQuery` object